docs/authoring.md

```diff
@@ -86,78 +86,132 @@ To make your component available as a NuGet package, it is important to include
+      Follow these steps to add an application manifest file to your project:
 
-### Consuming from C# applications
+      1. In Visual Studio, right click on the project node under **Solution Explorer** and click **Add -> New Item**. Search for the **Text File** template and name your file `<MyNativeAppName>.exe.manifest`.
 
-Consuming a C#/WinRT component from C#/.NET apps is supported by both package reference or project reference. This scenario is equivalent to consuming any ordinary C# class library and does not involve WinRT activation in most cases.
+      2. Add your activatable class registrations to the manifest file. Below is an example with a C#/WinRT component called `MyAuthoredComponent` and activatable classes `WinRTComponent.Class*`. Note that `WinRT.Host.dll` must be specified as the file name, without a relative path.
 
-**Note:** With C#/WinRT 1.3.5 or later, project references to a C#/WinRT component require .NET 6.
+            ```xml
+            <?xml version="1.0" encoding="utf-8"?>
+            <assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+              <assemblyIdentity version="1.0.0.0" name="CppConsoleApp"/>
+              <file name="WinRT.Host.dll">
+                <activatableClass name="MyAuthoredComponent.Class1"
+                                  threadingModel="both"
+                                  xmlns="urn:schemas-microsoft-com:winrt.v1" />
+                <activatableClass name="MyAuthoredComponent.Class2"
+                                  threadingModel="both"
+                                  xmlns="urn:schemas-microsoft-com:winrt.v1" />
+              </file>
+            </assembly>
+            ```
+
+      3. Modify the project to include the manifest file in the output when deploying the project. Right-click on the file in **Solution Explorer**, select **Properties**, and set the **Content** property to **True** using the drop-down arrow on the right.
+
+### Windows App SDK Applications
+
+Authoring and consuming WinUI types in Windows App SDK applications is in development and is supported with some exceptions.
+Component authors can implement WinUI types using the Windows App SDK C# class library project template. 
+Components can be consumed by application projects that reference the Windows App SDK.
 
-### Consuming from packaged applications
+#### Support Matrix
 
-Consuming C#/WinRT components from MSIX-packaged applications is supported with a few modifications.
+The following table outlines the Windows App SDK application types that are supported when consuming C#/WinRT authored components implementing WinUI types:
+
+- ✅ means the scenario works
+- 🟨 means the scenario partially works (i.e. with specific limitations)
+- 🟥 means the scenario does not work
+- Note: "WAP" refers to the usage of a separate Windows Application Packaging Project in packaged apps.
+
+|Application Type|Project or Package Reference|Supported?|

```

Comment:
I like the table approach though I'd recommend doing it as follows:

### C++

|  | Project Reference | Package Reference |
|:--|:-:|:-:|
| Packaged | ✅ | 🟨 |
| Packaged with WAP | 🟥 | 🟥 | 
| Unpackaged | 🟥 | 🟨 |


then another table for C# following it